### ISSUE TITLE
Add/remove period to end of context responses (alertive phrases) describing concentration level

### ISSUE THREAD

**Issue by terracoda** [phetsims/molarity#147]
@twant, I found an alert (context response) describing concetration level that does not end in a period.

The Context Responses indicating a Solute Change are fine - in fact, they are great!
The Context Responses once a solution is saturated are also fine.
It's just when the solution becomes not saturated and the alert ends with the concentraion level description.

You'll want to make sure you only add a period to the alert. The descriptions for concentration underlined in blue in the PDOM (second screenshot) do not want any periods.
![Screen Shot 2019-09-25 at 5 20 45 PM](https://user-images.githubusercontent.com/1902116/65634631-3c974080-dfb9-11e9-9b7c-7f6a93464185.png)

![Screen Shot 2019-09-25 at 5 07 11 PM](https://user-images.githubusercontent.com/1902116/65633903-ad3d5d80-dfb7-11e9-87c1-7566eb7a2b5d.png)

**Comment by twant** [comment]
@terracoda thanks! Fixed and double checked the other punctuation you pointed out to make sure none of it changed. I'm going to go ahead and close this one, since it's fairly small, but please let me know if you'd prefer to reopen and review!

**Comment by terracoda** [comment]
@twant, from master using phetmarks I just noticed two periods in the alert fires when I change  the selected solute with Solution Values checked. I am not sure if this showed up after the original issue was posted, so re-opening just in case.

![Screen Shot 2019-09-30 at 1 19 11 PM](https://user-images.githubusercontent.com/1902116/65894838-02e18380-e385-11e9-8253-95c4ec88d271.png)

And also when Solution Values is checked, the "Still saturated" alerts are missing periods.
![Screen Shot 2019-09-30 at 1 23 43 PM](https://user-images.githubusercontent.com/1902116/65895562-3cff5500-e386-11e9-97dd-d9240c9a5c35.png)
![Screen Shot 2019-09-30 at 1 26 33 PM](https://user-images.githubusercontent.com/1902116/65895573-3f61af00-e386-11e9-9e3a-f953d467a08c.png)
 
I think when Solution Values is not checked, everything is good.

**Comment by twant** [comment]
@terracoda @Matthew-phet pushed an update! Assigning back over to you to double check.

**Comment by terracoda** [comment]
You got it! Thanks!
